server/apps/main/templates/main/my_stories.html

```diff
@@ -0,0 +1,239 @@
+      padding: 0 15px;
+      margin: 20px 0;
+    }
+
+    /* End of application.html inline styles */
+
+    a.navbar-brand {
+      font-weight: 550 !important;
+    }
+
+    a:hover {
+      font-weight: 550 !important;
+      transition: 0.3s;
+    }
+
+    section.stories_page {
+      text-align: center;
+      margin-top: 30px;
+      color: #000;
+    }
+
+    div.story_head p.h3,
+    div.story_body p.h3 {
+      font-weight: 550;
+      font-size: 38px;
+      margin-bottom: 20px;
+    }
+
+    span.p_page {
+      font-weight: 500;
+      font-size: 19px;
+    }
+
+    div.story_head p.h5 {
+      font-size: 22px;
+      margin-top: 40px;
+      font-weight: 550;
+    }
+
+    ul.can_do {
+      list-style: none;
+      max-width: 600px !important;
+      margin: auto;
+    }
+
+    ul.can_do li.do {
+      font-size: 19px;
+      text-align: left !important;
+    }
+
+    div.story_body {
+      margin-top: 60px;
+    }
+
+    .btn {
+      padding: 6px 12px !important;
+      margin-top: 6px !important;
+      font-size: 14px;
+    }
+
+  </style>
+
+  <!-- Navbar -->
+  <div class="container">
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar"
+            aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+
+          <!-- Logo -->
+          <a class="navbar-brand" href="{% url 'main:index' %}">
+            AutSPACEs
+          </a>
+        </div>
+
+        <div id="navbar" class="navbar-collapse collapse">
+          <ul class="nav navbar-nav navbar-right">
+            <li {% if section == 'home' %} class="active" {%endif%}>
+              <a href="{% url 'main:index' %}">Home</a>
+            </li>
+            <li {% if section == 'main:list' %} class="active" {%endif%}>
+              <a href="{% url 'main:list' %}">View Experiences</a>
+            </li>
+            <li {% if section == 'main:my_stories' %} class="active" {%endif%}>
+              {% if not request.user.is_authenticated %}
+              <a href="{% url 'main:index' %}">Home</a>
+              {% else% }
+                <a href="">My Stories</a>
+              {% endif %}
+            </li>
+
+            {% if request.user.is_authenticated %}
+            {% comment %}
+              Couldn't find the share experiences url, 
+              so I'm leaving that there for anyone to fix it in 
+
+              {% if section == 'main:share-experiences-url %}
+
+              href="{% url 'main:share-experiences-url' %}"
+              
+            <li {% if section == '' %} class="active" {%endif%}>
+              <a href="{% url '' %}">Share Experiences</a>
+            </li>
```

Comment:
Totally okay that you couldn't find the share-experiences url. 

The button doesn't have to redirect anywhere, but can you make sure the button appears on the navbar? 
I think you only need to move the `{% endcomment %}` above this list item.

Reply:
Awesome! Yes, sure, I can do that.